webapp-django/crashstats/crashstats/static/crashstats/js/socorro/utils.js

```diff
@@ -230,7 +230,23 @@
                 }
 
                 return params;
-            }
+            },
+            sortResults: function (results, container, query) {
+                if (query.term) {
+                    return results.sort(function (a, b) {
+                        if (a.text.length > b.text.length) {
```

Comment:
Because I want to show the shortest matching result first. That is, I found, the best way to sort these field names.